config/webhook/llmisvc/service.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: llmisvc-webhook-server-service
+  name: llmisvc-webhook-svc
```

Comment:
Kustomize use `llmisvc-webhook-server-service` but why it is using different name?